### PR TITLE
Eliot recipe: a Python logging library.

### PR DESCRIPTION
**Pull request by itamarst**

### CODE REVIEW

**Comment by itamarst** [comment]
@conda-forge/help-python

File: recipes/eliot/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 973e5c9518b24a85d4ed2841882e32359ab5cdaf8389fc98dbd24866f052b84c
+
+build:
+  number: 0
+  skip: true  # [py<35]
```
Comment (beckermr) [inline recipes/eliot/meta.yaml]: If you remove this selector and turn it into a python version constraint, I think you make this package noarch: python. See the conda-forge documentation here: https://conda-forge.org/docs/meta.html#building-noarch-packages

Comment (itamarst) [reply recipes/eliot/meta.yaml]: Thanks, will check it out.

Comment (itamarst) [reply recipes/eliot/meta.yaml]: OK, done.

File: recipes/eliot/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  sha256: 973e5c9518b24a85d4ed2841882e32359ab5cdaf8389fc98dbd24866f052b84c
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python > 3.4
```
Comment (beckermr) [inline recipes/eliot/meta.yaml]: I am like 90% sure you need `python >3.4` here. I have gotten errors when the space is there for whatever reason.

Comment (itamarst) [reply recipes/eliot/meta.yaml]: Thanks, fixed.

**Comment by itamarst** [comment]
Ugh, apparently I can't do noarch because of setuptools' based scripts? Will fix in morning.

**Comment by beckermr** [comment]
Yup that is right. You don't have to fix it. We are happy to ship the package as is. But if you want to, I won't stop you. Noarch packages definitely help reduce CI usage which is an issue for conda-forge.

**Comment by itamarst** [comment]
So if shipped like this, the impact will be... no executables on windows?

**Comment by beckermr** [comment]
Just remove the noarch and it will all work fine.

**Comment by itamarst** [comment]
Done.

**Comment by itamarst** [comment]
Sigh. `OSError: [WinError 123] The filename, directory name, or volume label syntax is incorrect: 'C:\\bld\\eliot_1547817367357\\work' -> 'C:\\bld\\eliot_1547817367357\\work_moved_eliot-1.6.0-py>34_0_win-64_main_build_loop'`

I guess that's why they have skip syntax.

**Comment by itamarst** [comment]
OK, ready to go!

**Pull request merged**